.github/workflows/run_tests_scraper.yml

```diff
@@ -0,0 +1,81 @@
+name: Run district scrapers
```

Comment:
This should be something like "Run tests scraper"

Reply:
You're right! Should be fixed in https://github.com/openZH/covid_19/pull/1342/commits/f7b01d50ef3f5580f657dc40efc69ec2b5d17c65